pom.xml

```diff
@@ -332,7 +332,7 @@
     <version.org.rhq>4.8.0</version.org.rhq>
     <!-- Avoid using scannotation in favor of its successor org.reflections -->
     <version.org.scannotation>1.0.3</version.org.scannotation>
-    <version.org.slf4j>1.7.25</version.org.slf4j>
+    <version.org.slf4j>1.7.7</version.org.slf4j>
```

Comment:
This was bumped on purpose to be in sync with logback. I don't think we should downgrade.

Reply:
@psiroky  ok - I will rollback this

Reply:
@psiroky DONE